src/ExpressionEvaluator/Core/Source/ResultProvider/Formatter.cs

```diff
@@ -71,23 +73,23 @@ string IDkmClrFullNameProvider.GetClrTypeName(DkmInspectionContext inspectionCon
             return sawInvalidIdentifier ? null : name;
         }
 
-        string IDkmClrFullNameProvider.GetClrArrayIndexExpression(DkmInspectionContext inspectionContext, int[] indices)
+        string IDkmClrFullNameProvider.GetClrArrayIndexExpression(DkmInspectionContext inspectionContext, string[] indices)
```

Comment:
Is there a test somewhere for this? The debugger will call this with:
GetClrArrayIndexExpression(inspectionContext, new string[] {  string.Empty })

It would be good to add a test for string.Empty